docs-markdown/viz.md

```diff
@@ -0,0 +1,284 @@
+                          radius_stops = radius_stops,
+                          center = (-95, 40),
+                          zoom = 3,
+                          below_layer = 'waterway-label'
+                        )
+
+viz.show()
+```
+![screen shot 2018-02-21 at 3 34 18 pm](https://user-images.githubusercontent.com/11286381/36511755-b58e6cc8-171c-11e8-9385-e90c3795be14.png)
+
+
+## class HeatmapViz
+
+The `HeatmapViz` object handles the creation of a heat map and is built on top of the `MapViz` class.
+
+### Params
+**HeatmapViz**(_data, weight_property=None, weight_stops=None, color_stops=None, radius_stops=None, *args, **kwargs_)
+
+Parameter | Description | Example
+--|--|--
+data | name of GeoJson file or object
+weight_property | property to determine heatmap weight. | "population"
+weight_stops | stops to determine heatmap weight. | [[10, 0], [100, 1]]
+color_stops | stops to determine heatmap color. | [[0, "red"], [0.5, "blue"], [1, "green"]]
+radius_stops | stops to determine heatmap radius based on zoom. | [[0, 1], [12, 30]]
+
+
+### Methods
+**HeatmapViz.add_unique_template_variables**(_self, options_)
+Update map template variables specific to heatmap visual
+
+[View options](https://github.com/mapbox/mapboxgl-jupyter/blob/add-markdown-docs/docs-markdown/viz.md#params-4)
+
+### Usage
+```python
+import pysal.esda.mapclassify as mapclassify
+import pandas as pd
+import os
+from mapboxgl.utils import *
+from mapboxgl.viz import *
+
+# create pandas df from csv file
+df = pd.read_csv('points.csv')
+
+# Must be a public token, starting with `pk`
+token = os.getenv('MAPBOX_ACCESS_TOKEN')
+
+# Create a geojson file export from a Pandas dataframe
+df_to_geojson(df, filename='points.geojson',
+              properties=['Avg Medicare Payments', 'Avg Covered Charges', 'date'],
+                     lat='lat', lon='lon', precision=3)
+
+#Create a heatmap
+heatmap_color_stops = create_color_stops([0.01,0.25,0.5,0.75,1], colors='RdPu')
+heatmap_radius_stops = [[0,1], [15, 40]] #increase radius with zoom
+
+color_breaks = mapclassify.Natural_Breaks(df['Avg Medicare Payments'], k=8, initial=0).bins
+color_stops = create_color_stops(color_breaks, colors='Spectral')
+
+heatmap_weight_stops = create_weight_stops(color_breaks)
+
+#Create a heatmap
+viz = HeatmapViz('points.geojson',
+                  access_token=token,
+                  weight_property = "Avg Medicare Payments",
+                  weight_stops = heatmap_weight_stops,
+                  color_stops = heatmap_color_stops,
+                  radius_stops = heatmap_radius_stops,
+                  opacity = 0.9,
+                  center = (-95, 40),
+                  zoom = 3,
+                  below_layer='waterway-label'
+                )
+
+viz.show()
+```
+![screen shot 2018-02-21 at 3 34 55 pm](https://user-images.githubusercontent.com/11286381/36511775-cfc4d794-171c-11e8-86b9-5f1a6060a387.png)
+
+
+## class MapViz
```

Comment:
Let's put this class first in the docs since it's the parent class for all of the CircleViz, HeatmapViz, etc classes.